src/packages/cli/src/args.ts

```diff
@@ -281,9 +281,17 @@ export default function (
             }
           )
           .version(false);
+      },
+      function () {
+        // this handler executes when `ganache instances` is called without a subcommand
+        const command = chalk`{hex("${TruffleColors.porsche}") ganache instances}`;
+        console.log(`${command} requires a subcommand:`);
+        console.log();
+        yargs.showHelp();
+        yargs.exit(1, new Error("No subcommand provided"));
```

Comment:
Not my favourite. I'd rather use the `.demandCommand()` function, but this doesn't allow us to output the `instances` helptext, which I think is super useful.

This is also not easily testable, as it terminates the process inside of yargs :(